### DISCUSSION TITLE
Shorten variable descriptions

### DISCUSSION THREAD

**Discussion by IlanCosman** [PatrickF1/fzf.fish#243] [Archive] [answered]
Currently variables are described something like `set in global scope, exported, with 1 elements`
I propose instead: `global, exported, 1 elements`

For one thing, it just makes sense. It's a bit silly to be saying "set in X scope". Obviously the variable is set or we wouldn't be seeing it, and the whole "scope" part is redundant for anyone who understand the slightest bit about fish variables. After all we don't say "exported to child processes", we just say "exported".

It also has the great benefit of making the line not wrap around in a standard 80x24 terminal:

### Current: 
![image](https://user-images.githubusercontent.com/6318351/172749464-9a3b04db-f0a2-4c5c-b720-4b0218a8dddd.png)

### Proposed:
![image](https://user-images.githubusercontent.com/6318351/172749575-8f994db8-37a3-4367-938b-f5842388d68c.png)

I think that looks way nicer.

**Comment by PatrickF1** [comment]
Sure that's a good idea. Looks like you already have the code ready?

**Reply by IlanCosman** [reply]
Oh yah, I did, but I threw it away by accident 😅 Tbh I don't really understand what you're doing with the regexes in `_fzf_extract_var_info`, so I think it'd be best if I left it to you 😄 My version has some small bugs I think.

**Comment by PatrickF1** [comment, marked as answer]
Sorry I changed my mind. I'd rather have simpler code and make fewer assumptions about `set --show`'s brittle output (it's changed on me before).

> It also has the great benefit of making the line not wrap around in a standard 80x24 terminal

Btw how is a 80x24 a "standard" terminal size?

**Reply by IlanCosman** [reply]
> Sorry I changed my mind.

Ok 👍

> Btw how is a 80x24 a "standard" terminal size?

It just is 🤷‍♂️ You can search up "default terminal size" on google to get a bunch of sources. Seems to be for historical reasons.

**Reply by kidonng** [reply]
Terminals used to be screens for real, so have a standard size. If you assume any size matter _now_ you are doomed. Not to mention integrated terminals in editors (e.g. VS Code/JetBrains) are _way_ more wider by default. Tiling WM users have terminals full screen. And for any serious terminal usage a larger (wider) viewport is just more convenient and informative.

**Reply by PatrickF1** [reply]
This is super random and unrelated, but does that mean help message or man pages should be kept to 80 chars width?
